MyHeartCounts.xcodeproj/project.pbxproj

```diff
@@ -1188,6 +1203,14 @@
 				minimumVersion = 1.2.5;
 			};
 		};
+		8043E0092DC2348600E0DF01 /* XCRemoteSwiftPackageReference "swift-markdown-ui" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/gonzalezreal/swift-markdown-ui.git";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 2.4.1;
+			};
+		};
```

Comment:
We should probably deprecate our Markdown view in SpeziViews and just point to this package right away? Might not even require a view from us and Consent and other modules could use this then.

Reply:
i was thinking this as well. my plan would be to use the third-party MarkdownView a bit in MHC, see how well it works and if there are any caveats, and then adjust SpeziViews accordingly